expertise/create_dataset.py

```diff
@@ -300,12 +300,50 @@ def retrieve_expertise(self):
 
         return expertise
 
+    def get_papers_from_group(self, submission_group):
+        client = self.openreview_client
+        group_a_members = client.get_group(submission_group).members
+        pbar = tqdm(total=len(group_a_members), desc=submission_group + ' status...')
+        publications_by_profile_id = {}
+        all_papers = []
+
+        def get_status(profile_id):
+            pbar.update(1)
+            profile = openreview.tools.get_profile(client, profile_id)
+            if profile:
```

Comment:
Handle the case if there's no profile

Reply:
What do you suggest happens when there is no profile? Throw an exception? Admittedly, this part of the code is something Carlos wrote for ICLR so I'm not too familiar with these sections of code.

Reply:
if there is no profile then ignore it, there won't be scores for that user.

Reply:
When there's no profile, `get_status` will return `None`, which will cause problems later on when you try to access results from this `None` object. (line 330)

Reply:
@haroldrubio did you fix this?

Reply:
Yes, this function gets called here:
https://github.com/openreview/openreview-expertise/blob/454706ba16972a8d9bbaee8e066a9f1531fc5dd0/expertise/create_dataset.py#L324-L326

so `group_a_members` gets set here:
https://github.com/openreview/openreview-expertise/blob/454706ba16972a8d9bbaee8e066a9f1531fc5dd0/expertise/create_dataset.py#L308-L311

which uses `get_profile_ids()` defined here which guarantees that the members returned in the first element of the tuple have profiles:
https://github.com/openreview/openreview-expertise/blob/454706ba16972a8d9bbaee8e066a9f1531fc5dd0/expertise/create_dataset.py#L149